scripts/get_all_heron_paths.sh

```diff
@@ -112,7 +112,7 @@ function collect_generated_paths() {
   # uniq to avoid doing blaze query on duplicates.
   for path in $(find bazel-genfiles/ -name "*.java" | sed 's|/\{0,1\}bazel-genfiles/\{1,2\}|//|' | uniq); do
     source_path=$(echo ${path} | sed 's|//|bazel-genfiles/|' | sed 's|/com/.*$||')
-    echo "$(get_containing_library ${path}):${source_path}"
+#    echo "$(get_containing_library ${path}):${source_path}"
```

Comment:
It looks like this is the output of the for loop, which calls the removed functions. I don't really know this part of the code base but it might be useful I feel.